src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/RuntimeEnvironmentTest.cs

```diff
@@ -13,9 +13,16 @@ namespace System.Runtime.InteropServices
     public static class RuntimeEnvironmentTest
     {
         [Fact]
-        public static void RuntimeEnvironmentPosTest()
+        public static void RuntimeEnvironmentRuntimeDirectory()
         {
             Assert.True(Directory.Exists(RuntimeEnvironment.GetRuntimeDirectory()));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot,"RuntimeEnvironmentSysVersion not supported on UapAot")]
```

Comment:
I doubt that it's useful for AOT. In theory you could return the app directroy but I don't think that's what people want. Perhaps return null? @jkotas